javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java

```diff
@@ -21,32 +21,34 @@
 
 package com.github.javaparser;
 
-import java.io.IOException;
-import java.util.Collections;
-import java.util.LinkedList;
-import java.util.List;
-
 import com.github.javaparser.ast.CompilationUnit;
 import com.github.javaparser.ast.Node;
 import com.github.javaparser.ast.comments.Comment;
 import com.github.javaparser.ast.comments.CommentsCollection;
 import com.github.javaparser.ast.comments.CommentsParser;
 import com.github.javaparser.ast.comments.LineComment;
 
+import java.io.IOException;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
 /**
+ * Assigns comments to nodes of the AST.
+ * 
  * @author Sebastian Kuerten
  * @author Júlio Vilmar Gesser
  */
-public class CommentsInserter
-{
-
+class CommentsInserter {
     private boolean doNotAssignCommentsPreceedingEmptyLines = true;
     private boolean doNotConsiderAnnotationsAsNodeStartForCodeAttribution = false;
+    
+    CommentsInserter() {
+    }
 
-    public void insertComments(CompilationUnit cu, String code)
-            throws IOException {
+    public void insertComments(CompilationUnit cu, String comments) throws IOException {
```

Comment:
It is not 100% clear to me what comments should contain: just the text of the comments? All the original code?